### ISSUE TITLE
[Screen Reader] Close button name is not announced in Blocks hint when navigating via keyboard

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#6956]
**Describe the bug**
When Windows 11 Narrator is enabled, the Close button in the Blocks hint is not correctly announced during keyboard navigation in the Blocks Start Tutorial.

**Steps to reproduce the behavior:**
1. Enable **Windows 11 Narrator**.
2. Navigate to [Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta).
3. Use **Tab** to focus on the "**Blocks Start Tutorial**" button and press Enter.
4. Press **Tab** to focus on the hint icon and press **Enter** to open it.
5. Press **Tab** to move focus to the **Close** button.
6. Observe Narrator output.

**Actual behavior**
Narrator announces only "button" and does not read the Close button name.
 
**Expected behavior**
Narrator should announce the Close button with its accessible name when focused.

**Screen Recording**
https://github.com/user-attachments/assets/eaa27598-c53e-49dc-bc7e-65a5c6393530

**Additional context**
1.OS: Windows 11 25H2
2.Browser: Chrome, Edge, and Firefox
3.makecode.microbit.org version: 8.1.26
4.Microsoft MakeCode version: 12.3.21

**Comment by microbit-matt-hillsdon** [comment]
The button lacks an accessible name:

```
<button class="ui button icon  tutorial-callout-close" tabindex="0"><i class="icon close " aria-hidden="true" role="presentation"></i></button>
```
